lib/app_web/router.ex

```diff
@@ -25,6 +25,12 @@ defmodule AppWeb.Router do
     resources "/status", StatusController
     resources "/tags", TagController
     resources "/timers", TimerController
+
+    # capture
+    resources "/capture", CaptureController, only: [:new, :create]
```

Comment:
thanks for adding this. 👍 
it has reminded me of what we want the **`default`** route to be ... 
https://github.com/dwyl/app/issues/246
(_detail will be added ahead of "Sprint 1" next week_)